js/components/appcues.tsx

```diff
@@ -0,0 +1,45 @@
+import { getMetaContent } from "../util/metadata";
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+declare global {
+  const Appcues: {
+    identify(
+      userId: string,
+      properties?: Record<string, string | number | boolean>,
+    ): void;
+    page(): void;
+  };
+}
+
+// React component, put inside react router
+export const AppcuesTrackPage = (): null => {
+  const email = getMetaContent("userEmail") ?? "";
+  const appcuesUserId = getMetaContent("appcuesUserId");
```

Comment:
Doesn't `getMetaContent` have an allowlist of known keys? I'm surprised that this PR doesn't have an edit to `js/util/metadata.ts`. Why doesn't that cause typechecking to fail?

Reply:
Ah, I missed the commit that added that by like a minute.